backend/src/services/calendar/database/operations.py

```
@@ -0,0 +1,1758 @@
+            if master.end_datetime and master.start_datetime:
+                duration = master.end_datetime - master.start_datetime
+            
+            try:
+                instance_dates = expand_recurrence(
+                    recurrence=master.recurrence,
+                    start=start_dt,
+                    time_min=min_dt,
+                    time_max=max_dt,
+                    max_instances=max_results,
+                )
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)